packages/angular/projects/clr-angular/src/forms/common/form.ts

```diff
@@ -20,8 +20,9 @@ import { ClrLabel } from './label';
 })
 export class ClrForm {
   @Input('clrLabelSize')
-  set labelSize(size: number) {
-    this.layoutService.labelSize = size;
+  set labelSize(size: number | string) {
+    const sizeNumber = parseInt(size as string, 10) || 0;
```

Comment:
We should default to the default label size (2 I believe) otherwise this would generate an invalid label size of 0.

Reply:
Good catch! I've updated it to `2`.